packages/core/admin/admin/src/content-manager/pages/ListView/utils/buildValidQueryParams.js

```diff
@@ -0,0 +1,32 @@
+import set from 'lodash/set';
+
+import createPluginsFilter from './createPluginsFilter';
+
+/**
+ * Creates a valid query params object
+ * This includes:
+ * - a filters clause
+ * - plugin options
+ */
+const buildValidQueryParams = (queryParams = {}) => {
+  /**
+   * Extracting pluginOptions from the query since we don't want them to be part
+   * of the url
+   */
+  const {
+    plugins: _,
+    _q: query,
+    ...otherQueryParams
+  } = {
+    ...queryParams,
+    ...createPluginsFilter(queryParams.plugins),
+  };
+
+  if (query) {
+    set(otherQueryParams, `_q`, encodeURIComponent(query));
```

Comment:
How come you feel the need to use `set`? I think we can avoid lodash here :)

Reply:
This was just a quick and dirty copy paste of buildQueryString to get things working. I'll take a look at removing set.